dashboard/app/dsl/script_dsl.rb

```diff
@@ -45,6 +46,10 @@ def trophies(trophies_string)
     @trophies = ActiveRecord::ConnectionAdapters::Column::value_to_boolean(trophies_string)
   end
 
+  def wrapup_video(wrapup_video)
```

Comment:
Waiting to do this cleanup as part of a separate change.  This change is already somewhat risky so I'd prefer to keep it bare-bones.